pixart.js

```diff
@@ -1 +1,30 @@
+$(document).ready(function () {
 
+function changeColor(color) {
+  $(".brush").css("background-color",color)
+}
+
+$("#set-color")[0].addEventListener("click", function(event){
```

Comment:
I agree, but the instructions on the assignment called for using both together for some reason.